src/main/java/core/basesyntax/DataConverterImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : data) {
+            if (line.startsWith("type")) {
+                continue;
```

Comment:
Avoid using 'magic values' such as 'type'. It would be better to define it as a constant to give it a meaningful name and to avoid mistakes when using it multiple times.